itests/utilities/issue_tx_utility.go

```diff
@@ -0,0 +1,51 @@
+package utilities
+
+import (
+	"time"
+
+	"github.com/wavesplatform/gowaves/itests/config"
+	i "github.com/wavesplatform/gowaves/itests/fixtures"
+	"github.com/wavesplatform/gowaves/itests/net"
+	"github.com/wavesplatform/gowaves/pkg/crypto"
+	"github.com/wavesplatform/gowaves/pkg/proto"
+)
+
+func GetAccount(suite *i.BaseSuite, i int) config.AccountInfo {
+	return suite.Cfg.Accounts[i]
+}
+
+func GetAvalibleBalanceInWavesGo(suite *i.BaseSuite, address proto.WavesAddress) int64 {
+	return suite.Clients.GoClients.GrpcClient.GetWavesBalance(suite.T(), address).GetAvailable()
+}
+
+func GetAssetBalanceGo(suite *i.BaseSuite, address proto.WavesAddress, id []byte) int64 {
+	return suite.Clients.GoClients.GrpcClient.GetAssetBalance(suite.T(), address, id).GetAmount()
+}
+
+func GetTxIdsInBlockchain(suite *i.BaseSuite, ids map[string]*crypto.Digest, timeout time.Duration) map[string]string {
+	time.Sleep(timeout)
```

Comment:
Btw, it's ok to have `Sleep` here? Looks like unnecessary slowdown of test. You can wait and range through all in parallel.